src/main/java/de/doubleslash/keeptime/view/ProjectReport.java

```diff
@@ -41,14 +41,22 @@ public ProjectReport(final int size) {
 
    public void appendToWorkNotes(final String currentWorkNote) {
       this.numberOfNotes++;
+      this.sb.append(currentWorkNote.trim());
```

Comment:
not sure why you append something and one line later we check if it is empty? so we do add also empty strings?